decidim-proposals/app/forms/decidim/proposals/admin/proposal_base_form.rb

```diff
@@ -82,7 +82,7 @@ def geocoded?
         # Finds the Meetings of the current participatory space
         def meetings
           @meetings ||= Decidim.find_resource_manifest(:meetings).try(:resource_scope, current_component)
-                               &.order(title: :asc)
+                          &.order(title: :asc)&.published
```

Comment:
Does this safe navigation operator makes sense before published? `.order` can return nil?

Reply:
Good catch!